protoc-gen-openapiv2/main.go

```diff
@@ -43,6 +43,7 @@ var (
 	disableServiceTags             = flag.Bool("disable_service_tags", false, "if set, disables generation of service tags. This is useful if you do not want to expose the names of your backend grpc services.")
 	disableDefaultResponses        = flag.Bool("disable_default_responses", false, "if set, disables generation of default responses. Useful if you have to support custom response codes that are not 200.")
 	useAllOfForRefs                = flag.Bool("use_allof_for_refs", false, "if set, will use allOf as container for $ref to preserve same-level properties.")
+	allowPatchFeature              = flag.Bool("allow_patch_feature", true, "determines whether to use PATCH feature involving update masks (using google.protobuf.FieldMask).")
```

Comment:
This help text is not great. We should tailor it to this generator. What do you think about
```suggestion
	allowPatchFeature              = flag.Bool("allow_patch_feature", true, "whether to hide update_mask fields from the generated swagger file.")
```

Reply:
I mistakenly thought I needed to keep the same help text